docs/APIReference-Container.md

```diff
@@ -448,3 +459,82 @@ module.exports = Relay.createContainer(ProfilePicture, {
 - `COLLISION_COMMIT_FAILED` — Transaction was queued for commit but another transaction with the same collision key failed. All transactions in the collision queue, including this one, have been failed. Transaction can be recommitted or rolled back.
 - `COMMITTING` — Transaction is waiting for the server to respond.
 - `COMMIT_FAILED` — Transaction was sent to the server for comitting but failed.
+
+## Static Methods
+
+### getFragment
+
+```
+getFragment(
+  fragmentName: string, 
+  variables?: {[name: string]: mixed}
+): RelayFragmentReference
+```
+
+Gets a reference to a child container's fragment for inclusion in a parent fragment.
+
+#### Example
+
+Fragment composition is achieved via ES6 template string interpolation and `getFragment`: 
+
+```{6}
+// Parent.js
+Relay.createContainer(Parent, {
+  fragments: {
+    parentFragment: () => Relay.QL`
+      fragment on Foo {
+        id
+        ${Child.getFragment('childFragment')}
+      }
+    `,
+  }
+});
+// Child.js
+Relay.createContainer(Child, {
+  initialVariables: {
+    count: 5,
+  },
+  fragments: {
+    childFragment: () => Relay.QL`
+      fragment on Foo {
+        friends(first: $count) { ... }
+      }
+    `,
+  }
+});
+```
+
+In this example, whenever `Parent` is fetched, `Child`'s fragment will also be fetched. When rendering, `<Parent>` will only have access to the `props.foo.id` field;  data from the child fragment will be masked [*masked*](http://facebook.github.io/relay/docs/thinking-in-relay.html#data-masking). By default, `childFragment` will use its corresponding initial variables. Relay will fetch `friends(first: 5)`. When `<Child>` is rendered it will also make the initial variables available as `props.relay.variables = {count: 5}`.
```

Comment:
I considered this, but `setVariables` is pretty well covered in the guides and API docs.